statistics-presences/src/main/resources/public/ts/filter/filter.ts

```diff
@@ -91,16 +92,25 @@ export class FilterTypeFactory {
         return new FilterValue(value, selected);
     }
 
-    public changeUnProvingReasons(value: boolean): void {
-        this.reasons.forEach(reason => {
-            if (reason.proving === false) this.reasonsMap[reason.id] = value;
-        });
+    public changeUnProvingAbsences(value: boolean): void {
+        this.reasons.filter((reason: Reason) => reason.reason_type_id == REASON_TYPE_ID.ABSENCE)
+            .forEach((reason: Reason) => {
+                if (reason.proving === false) this.reasonsMap[reason.id] = value;
```

Comment:
this.reasons.filter((reason: Reason) => reason.reason_type_id == REASON_TYPE_ID.ABSENCE && reason.proving === false)
            .forEach((reason: Reason) => {
                this.reasonsMap[reason.id] = value;

c'est un peu plus propre